_posts/2023-05-10-solidity-0.8.20-release-announcement.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.20 Release Announcement'
+date: '2023-05-10'
+author: Solidity Team
+category: Releases
+---
+
+
+We are excited to announce the latest release of the Solidity Compiler, [Solidity v0.8.20](https://github.com/ethereum/solidity/releases/tag/v0.8.20). 
+This latest version includes a range of improvements and, most importantly, support for Shanghai!
+It also contains performance improvements in the via-IR pipeline and improves the list of events exposed in the contract ABI.
+
+### Important Note
+
+This compiler switches the **default** target EVM version to Shanghai, which means that the generated bytecode will include ``PUSH0`` opcodes.
+Be sure to select the appropriate EVM version in case you intend to deploy on a chain other than mainnet like L2 chains that may not yet support ``PUSH0``,
+otherwise deployment of your contracts will fail.
```

Comment:
```suggestion
This compiler switches the **default** target EVM version to Shanghai, which means that the generated bytecode will include ``PUSH0`` opcodes.
Be sure to [select the appropriate EVM](https://docs.soliditylang.org/en/latest/using-the-compiler.html#setting-the-evm-version-to-target) version in case you intend to deploy on a chain other than mainnet like L2 chains that may not yet support ``PUSH0``,
otherwise, deployment of your contracts will fail.
```

Added a link @ekpyron.